docker-compose.yml

```diff
@@ -57,9 +57,63 @@ services:
     networks:
       - celestia-zkevm-net
 
+  rollkit-evm-single:
+    image: ghcr.io/rollkit/rollkit-evm-single:v0.1.3
+    container_name: rollkit-evm-single
+    entrypoint: /usr/bin/entrypoint.sh
+    environment:
+      - EVM_ENGINE_URL=http://reth:8551
+      - EVM_ETH_URL=http://reth:8545
+      - EVM_JWT_SECRET=f747494bb0fb338a0d71f5f9fe5b5034c17cc988c229b59fd71e005ee692e9bf
+      - EVM_GENESIS_HASH=0x06bab65f0fa3c9b1ba0da34e319d7a5cd948cd10ba33080b88f17d281880e344
+      - EVM_BLOCK_TIME=1s
+      - EVM_SIGNER_PASSPHRASE=secret
+      - DA_ADDRESS=http://celestia-bridge:26658
+      - DA_AUTH_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.qxoN_a-k7aZZE77VSpfgyvI_0eNvJ_OgBsT35zxUQnQ
```

Comment:
Want to flag here, I have `--rpc.skip-auth` on celestia-node and verified by locally hitting the `node.Ready` jsonrpc. 

Still, without this env variable I got the following, even tho this token doesn't correspond to the account being used in the bridge node.

```
rollkit-evm-single  | 12:36PM ERR RPC call failed error="RPC client error: sendRequest failed: request failed, http status 401 Unauthorized" method=MaxBlobSize module=main
```

Reply:
Any idea why this is the case? cc. @Manav-Aggarwal